extensions/sql-migration/src/constants/helper.ts

```diff
@@ -264,6 +264,11 @@ export function canDeleteMigration(migration: DatabaseMigration | undefined): bo
 }
 
 export function canRetryMigration(migration: DatabaseMigration | undefined): boolean {
```

Comment:
I know you explained to us the new meaning of retry vs restart, but it might not be immediately obvious to people looking at it for the first time. Can we add some comments briefly explaining the difference?